client/package.json

```diff
@@ -19,11 +19,11 @@
     "axios": "^1.4.0",
```

Comment:
I think we can remove this dependency now!

Reply:
I can't reply to your other comment so responding to everything here!
- `pages` will be deleted when we merge this PR (it just looks strange on GH)
- Just pushed new commits to get rid of `NavBar` and `axios` 
- Next doesn't specify naming conventions, so we are free to do whatever we want! I'm fine to change our component filenames to PascalCase if we want to though!

Reply:
Sounds good! I'm fine with the naming convention we have now, just wanted to make sure it wasn't something we needed to keep track of with the new App Router. Thank you for answering all my questions!

Reply:
Of course! Just chilling here on the bus happy to have something to do